docs/embeddings/amazon-bedrock.md

```diff
@@ -0,0 +1,66 @@
+---
+---
+
+# Amazon Bedrock
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<div class="select-language">Select a language</div>
+
+<Tabs queryString groupId="lang">
+<TabItem value="py" label="Python"></TabItem>
+<TabItem value="js" label="JavaScript"></TabItem>
+</Tabs>
+
+
+Chroma provides a convinient wrapper for [Amazon Bedrock](https://aws.amazon.com/bedrock/) embedding API. This embedding function runs remotely on Amazon's servers, and requires an Amazon client information.
+
+<Tabs queryString groupId="lang" className="hideTabSwitcher">
+<TabItem value="py" label="Python">
+
+To use Amazon Bedrock embedding API, you must have `boto3` Python package installed and create an instance of `boto3.Session` with your AWS credentials. To use:
```

Comment:
@chezou, is it beneficial to also add in parenthesis how to install the package, e.g. (`pip install boto3`)

Reply:
Added https://github.com/chroma-core/docs/pull/204/commits/7390e294143b62aeeb2e8f62c535f17a1c62726b